package/ti-k3-boot-firmware-next/ti-k3-boot-firmware.mk

```diff
@@ -4,38 +4,16 @@
 #
 ################################################################################
 
-TI_K3_BOOT_FIRMWARE_NEXT_VERSION = 09.02.00.010
+TI_K3_BOOT_FIRMWARE_NEXT_VERSION = 10.00.06
 TI_K3_BOOT_FIRMWARE_NEXT_SITE = https://git.ti.com/cgit/processor-firmware/ti-linux-firmware/snapshot
 TI_K3_BOOT_FIRMWARE_NEXT_SOURCE = ti-linux-firmware-$(TI_K3_BOOT_FIRMWARE_NEXT_VERSION).tar.gz
 TI_K3_BOOT_FIRMWARE_NEXT_INSTALL_IMAGES = YES
-TI_K3_BOOT_FIRMWARE_NEXT_INSTALL_TARGET = YES
 TI_K3_BOOT_FIRMWARE_NEXT_LICENSE = TI Proprietary
 TI_K3_BOOT_FIRMWARE_NEXT_LICENSE_FILES = LICENSE.ti
 
-UBOOT_PRE_BUILD_HOOKS += TI_K3_BOOT_FIRMWARE_NEXT_RDEPEND_CMD
-
-define TI_K3_BOOT_FIRMWARE_NEXT_RDEPEND_CMD
-	$(MAKE) ti-k3-boot-firmware -C $(BASE_DIR)
-endef
-
-TI_DM_FW_DIR = ti-dm/am62xx
-TI_DM_FW_FILE = ipc_echo_testb_mcu1_0_release_strip.xer5f
-
-UBOOT_MAKE_OPTS += DM=$(BINARIES_DIR)/$(TI_DM_FW_DIR)/$(TI_DM_FW_FILE)
```

Comment:
:question: uboot no longer requires the device-manager (DM) firmware? or is it being configured elsewhere now?

Reply:
It's all been removed from upstream Buildroot:
https://github.com/buildroot/buildroot/blob/27e7b0bf14cf2321e2aae64e490671f5cfa2281d/boot/ti-k3-boot-firmware/ti-k3-boot-firmware.mk